packages/vuelidate/src/core.js

```diff
@@ -437,13 +442,13 @@ export function setValidations ({ validations, state, key, parentKey, childResul
    * If we have no `key`, this is the top level state
    * We dont need `$model` there.
    */
-  let $model = key ? '' : computed({
+  let $model = key ? computed({
```

Comment:
This was broken. Now it works per old Vuelidate spec, all tree levels have a $model. The top one is just null in these cases.